[2025-05-19T07:55:40.968+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: math_crawler_pipeline.crawl_and_save manual__2025-05-19T07:55:37.864911+00:00 [queued]>
[2025-05-19T07:55:40.975+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: math_crawler_pipeline.crawl_and_save manual__2025-05-19T07:55:37.864911+00:00 [queued]>
[2025-05-19T07:55:40.976+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2025-05-19T07:55:40.976+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 3
[2025-05-19T07:55:40.976+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2025-05-19T07:55:40.986+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): crawl_and_save> on 2025-05-19 07:55:37.864911+00:00
[2025-05-19T07:55:40.990+0000] {standard_task_runner.py:55} INFO - Started process 889 to run task
[2025-05-19T07:55:40.993+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'math_crawler_pipeline', 'crawl_and_save', 'manual__2025-05-19T07:55:37.864911+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/MathDag.py', '--cfg-path', '/tmp/tmpvjm4eh_o']
[2025-05-19T07:55:40.995+0000] {standard_task_runner.py:83} INFO - Job 8: Subtask crawl_and_save
[2025-05-19T07:55:41.043+0000] {task_command.py:389} INFO - Running <TaskInstance: math_crawler_pipeline.crawl_and_save manual__2025-05-19T07:55:37.864911+00:00 [running]> on host 23e48ed0cd15
[2025-05-19T07:55:41.087+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=math_crawler_pipeline
AIRFLOW_CTX_TASK_ID=crawl_and_save
AIRFLOW_CTX_EXECUTION_DATE=2025-05-19T07:55:37.864911+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-05-19T07:55:37.864911+00:00
[2025-05-19T07:55:41.097+0000] {MathDag.py:36} WARNING - No links found!
[2025-05-19T07:55:41.097+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-05-19T07:55:41.103+0000] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=math_crawler_pipeline, task_id=crawl_and_save, execution_date=20250519T075537, start_date=20250519T075540, end_date=20250519T075541
[2025-05-19T07:55:41.131+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2025-05-19T07:55:41.143+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
